js/id/ui/raw_tag_editor.js

```diff
@@ -85,7 +85,11 @@ iD.ui.RawTagEditor = function(context) {
         $items.order();
 
         $items.each(function(tag) {
-            var reference = iD.ui.TagReference({key: tag.key, value: tag.value}, context);
+            var reference;
+            if (id[0] === 'r' && tag.key === 'type')
```

Comment:
@tyrasd I'm ok with what you did, it's clear to me.  

I guess if you want to make the code a little clearer, you can do (before the loop)
```js
var isRelation = (context.entity(id).type === 'relation');
```